### PR TITLE
project_Details bug fix

### PR DESCRIPTION
**Pull request by raj454raj**
Ajax call breaks when submitting a record for staff.

### CODE REVIEW

**Comment by raj454raj** [comment]
@flavour: Just a sec merging one more bug :p

**Comment by nursix** [comment]
The project_Details bug fix is missing ;)

**Comment by nursix** [comment]
Ah - it's already in trunk, so it doesn't appear in the diff anymore. Ok ok

**Comment by raj454raj** [comment]
@nursix : Fran is too fast for a second :D :D

File: controllers/project.py [hunk 1]
```diff
@@ -1216,4 +1216,13 @@ def campaign_response_summary():
 
     return s3_rest_controller()
 
+# -----------------------------------------------------------------------------
+def human_resource_project():
```
Comment (nursix) [inline controllers/project.py]: This used to be the "human_resource" controller, which is still defined in this file but commented. Maybe just uncomment+rename, or remove the commented version and replace with this one?

Comment (raj454raj) [inline controllers/project.py]: @nursix : Done :+1:

Comment (nursix) [inline controllers/project.py]: Fine by me, good to merge.

**Pull request merged**